### ISSUE TITLE
Index_cpu.cgi and stats.cgi high cpu usage.

### ISSUE THREAD

**Issue by Vagrantin** [virtualmin/virtualmin-gpl#37]
I'm not running powerfull VM and I'm seeing ``Index_cpu.cgi`` and ``stats.cgi`` using most of my CPU (~40%).
Is it possible and ok to disable those modules ? 
If yes what would be the recommended way to do it ?

Thanks for any input.

Matth

**Comment by qooob** [comment]
Matth, hi.

This is ugly theme 19.05 bug that has been fixed on 19.06 already.

It only affects systems with disables system status collection.

Force Update the theme from settings and the load will drop to normal.

In case you still don't want to have real-time stats enabled, you can control it in theme's settings, under System Information tab at the very bottom.

**Comment by Vagrantin** [comment]
I'm already running 19.06 and it's still using way to much CPU for my config.
```
Kernel and CPU | Linux 4.9.0-4-amd64 on x86_64 | 
Processor information | Intel(R) Xeon(R) CPU E5-2620 v2 @ 2.10GHz, 2 cores
Real memory | 5.70 GB total / 547.64 MB used
```
So I've effectively disabled it in the Theme's settings.

Thanks for the help.
Matth
